chart/compass/templates/tests/director/director-test.yaml

```diff
@@ -43,6 +43,8 @@ spec:
               value: "https://{{ .Values.global.gateway.mtls.external.host }}.{{ .Values.global.ingress.domainName }}{{ .Values.global.director.prefix }}/graphql"
             - name: APP_DIRECTOR_EXTERNAL_CERT_FORMATION_MAPPING_ASYNC_URL
               value: "https://{{ .Values.global.gateway.mtls.external.host }}.{{ .Values.global.ingress.domainName }}{{ .Values.global.director.formationAsyncApi.pathPrefix }}{{ .Values.global.director.formationAsyncApi.path }}"
+            - name: APP_DIRECTOR_INTERNAL_URL
+              value: "https://{{ $.Values.global.gateway.tls.secure.internal.host }}.{{ $.Values.global.ingress.domainName }}{{ $.Values.global.director.prefix }}/graphql"
```

Comment:
Could we rename it `APP_DIRECTOR_INTERNAL_GATEWAY_URL` or something similar because with that name it seems to be "internal director" and it's an "external" through the gateway. There are two types of internal communication, so to say, internal(inside the cluster) and internal but the call is from the "outside" - istio, oathkeeper rules, etc. This seems to be the second one?

Reply:
I use this one for a new client in the tests using the service account token instead of the cert client. I've copied the env variable from another place in the tests. But if it makes more sense, it wouldn't be a problem to rename it.

Reply:
Maybe the other place that you've copied is not correctly named or perhaps it's a different case, nevertheless, such name differentiation between the two types of internal clients/URLs would be a nice thing to have.